beets/config_default.yaml

```diff
@@ -131,6 +131,7 @@ match:
         track_index: 1.0
         track_length: 2.0
         track_id: 5.0
+        style: 5.0
```

Comment:
It doesn't look like style is used for matching, so this is probably unnecessary.